pkg/api/v1beta4/dynakube/oneagent/spec.go

```diff
@@ -105,6 +105,11 @@ type HostInjectSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="OneAgent SecComp Profile",order=17,xDescriptors="urn:alm:descriptor:com.tectonic.ui:selector:core:v1:Namespace"
 	SecCompProfile string `json:"secCompProfile,omitempty"`
 
+	// Sotrage Host Path is the host filesystem to store configuration.
```

Comment:
```suggestion
	// StorageHostPath is the path on the host filesystem where OneAgent configurations will be stored.
```

cc @chrismuellner (you are better with descriptions 🙈 )

Reply:
fixed in 9bf77bf39

Reply:
we could specify `writable path` to make it clear that it's that, other than that I like marcell's suggestion

Reply:
fixed in f4ab4272e

Reply:
Please change to 
```
StorageHostPath is the writable directory on the host filesystem where OneAgent configurations will be stored.
```

Reply:
@chrismuellner fixed in 9a0648d76